official/v7/querybuilders/search_queries_aggs.go

```diff
@@ -0,0 +1,125 @@
+package querybuilders
+
+// AggsQuery is used to build an aggregation elastic query
+// For more details, see:
+// https://www.elastic.co/guide/en/elasticsearch/reference/7.0/search-aggregations.html
+type AggsQuery struct {
+	metricAggs              []MetricAggregation
+	bucketTermsAggs         []BucketTermsAggregation
+	bucketDateHistogramAggs []BucketDateHistogramAggregation
+}
+
+// Creates a new aggs query with empty aggregations lists
+func NewAggsQuery() *AggsQuery {
+	return &AggsQuery{
+		metricAggs:              make([]MetricAggregation, 0),
+		bucketTermsAggs:         make([]BucketTermsAggregation, 0),
+		bucketDateHistogramAggs: make([]BucketDateHistogramAggregation, 0),
+	}
+}
+
+// BucketTerms appends a MetricAggregation list to the metricAggs field
+func (q *AggsQuery) Metric(aggs ...MetricAggregation) *AggsQuery {
+	q.metricAggs = append(q.metricAggs, aggs...)
+	return q
+}
+
+// BucketTerms appends a BucketTermsAggregation list to the bucketTermsAggs field
+func (q *AggsQuery) BucketTerms(aggs ...BucketTermsAggregation) *AggsQuery {
+	q.bucketTermsAggs = append(q.bucketTermsAggs, aggs...)
+	return q
+}
+
+// BucketDateHistogram appends a BucketDateHistogramAggregation list to the bucketDateHistogramAggs field
+func (q *AggsQuery) BucketDateHistogram(aggs ...BucketDateHistogramAggregation) *AggsQuery {
+	q.bucketDateHistogramAggs = append(q.bucketDateHistogramAggs, aggs...)
+	return q
+}
+
+// Source is a helper function used to build elastic query
+// it returns a map that can be marshalled into json string
+func (q *AggsQuery) Source() (interface{}, error) {
+	// {
```

Comment:
Should this comments be here?

Reply:
Yes, it was on purpose. These comments exists to show the output of the Source function and it is an example of how the elastic search query will be.
Also, this type of comment is presente in every other Source function. (example: official/v7/querybuilders/search_queries_bool.go)

Reply:
Maybe we should enrich the text making that explicit, otherwise we risk it looking like code that was commented out